scripts/update/Updater.php

```diff
@@ -64,7 +64,7 @@ public function update($initialVersion)
             $serviceManager->register(DeliveryExecutionFinderService::SERVICE_ID, new DeliveryExecutionFinderService());
             $serviceManager->register(QtiRunnerInitDataBuilderFactory::SERVICE_ID, new QtiRunnerInitDataBuilderFactory());
 
-            $this->setVersion('0.6.0');
+            $this->setVersion('0.6.1');
```

Comment:
Please not borrow an existing update script to bump the version. Please create a new one each time!

Reply:
Usually, I do such a way, but it seems like rebase issue.